Lavinia-api/Data/CustomValidation.cs

```diff
@@ -42,18 +63,40 @@ public static void ValidateCountry(Country country, HashSet<int> checkedModels)
         /// <param name="checkedModels">A set of models that have already been checked.</param>
         private static void ValidateCounty(County county, HashSet<int> checkedModels)
         {
-            if (county == null) throw new ArgumentException("County cannot be null.");
+            if (county == null)
+            {
+                throw new ArgumentException("County cannot be null.");
+            }
+
+            if (county.Name == null)
+            {
+                throw new ArgumentException("County.Name cannot be null.");
+            }
 
-            if (county.Name == null) throw new ArgumentException("County.Name cannot be null.");
```

Comment:
These changes seem completely unrelated to the commit message.

Reply:
The git lines are misaligned, if you look at line 71 in the new version you will find the same if, but with curly braces.